test/testMongoCrudHandler.js

```diff
@@ -190,10 +190,11 @@ describe('metadb:', function () {
       });
     });
 
-    it('should fail to fetch if the password is bad', function (done) {
+    it('should be able to fetch even if the hash is bad as it is ignored', function (done) {
```

Comment:
so there is no longer a hash used  so should the test description be uploaded to reflect that fact